src/cascadia/TerminalApp/TerminalTab.h

```diff
@@ -7,6 +7,9 @@
 #include "TabBase.h"
 #include "TerminalTab.g.h"
 
+static constexpr uint32_t HeaderRenameBoxWidthDefault{ 165 };
+static constexpr uint32_t HeaderRenameBoxWidthTitleLength{ 1024 };

```

Comment:
Should the max width in SizeToContent mode be something like `NaN`, to _clear_ the max width, rather than set it arbitrarily large?

Reply:
This was the value of the max width before this change and I decided to just stick with that

Reply:
Er, that was the _max length_, in characters 😉